packages/react/src/OneSchemaImporter.tsx

```diff
@@ -97,7 +100,8 @@ export default function OneSchemaImporter(props: OneSchemaImporterProps) {
         importer.close()
       }
     }
-  }, [props.isOpen])
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
disabling this one because the things i'm using that aren't mentioned are all about the config & if someone for some reason changed the config while in the importer flow... rn i'm not sure we'd handle that gracefully, so, if they want the uptake they have to close and reopen